src/content/developers/docs/apis/json-rpc/index.md

```diff
@@ -13,7 +13,7 @@ JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol. Prim
 
 ## JSON-RPC resources {#json-rpc-resources}
 
-- [Ethereum JSON-RPC Specification](https://playground.open-rpc.org/)
+- [Ethereum JSON-RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ethereum/eth1.0-apis/assembled-spec/openrpc.json&uiSchema[appBar][ui:splitView]=false&uiSchema[appBar][ui:input]=false&uiSchema[appBar][ui:examplesDropdown]=false)
 - [Ethereum JSON-RPC Specification GitHub repo](https://github.com/ethereum/eth1.0-apis)
```

Comment:
Thanks for the PR @shanejonas 😀✌️

I think we had a typo here. We definitely want to link directly to the Playground - I didn't even know it exists until around 1559 and think it's useful to continue people to it. But I think it's worth keeping the link you have added also (I find it more digestible than the GitHub repo). What about doing this, instead?

```suggestion
- [Open-RPC Playground](https://playground.open-rpc.org/)
- [Ethereum JSON-RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ethereum/eth1.0-apis/assembled-spec/openrpc.json&uiSchema[appBar][ui:splitView]=false&uiSchema[appBar][ui:input]=false&uiSchema[appBar][ui:examplesDropdown]=false)
- [Ethereum JSON-RPC Specification GitHub repo](https://github.com/ethereum/eth1.0-apis)
```

Reply:
linking directly to the playground does not link to anything ethereum related, OpenRPC is a generic JSON-RPC definition language, its default page shows a pet store example, not sure thats useful to link from here:


![image](https://user-images.githubusercontent.com/364566/129779773-3c114158-1402-40ba-81f4-486cfeb58229.png)


you are seeing the eth spec when navigating there because its the last open spec in localstorage

Reply:
I think you're right @shanejonas - I think we want to replace this link.